### PR TITLE
Initial code commit

### PR DESCRIPTION
**Pull request by tbrebant**
Initial code commit.
Ping @jrouault and @jperdereau.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# cordova-plugin-webviewselector
+
+This plugin is designed to allow developers that are using Crosswalk with [cordova-plugin-crosswalk-webview](https://github.com/crosswalk-project/cordova-plugin-crosswalk-webview) to be able to switch dynamically between Crosswalk webview and default system webview (on Android).
+
+#### Why anyone would want that?
+
+Crosswalk being not maintained anymore, the future is in the use of system's webview. But on older devices we may want to keep a well working older version of Chromium. We may also want to let your application's users to switch themselves between engines for some reasons, like if your euristic to select the best engine automatically depending on the device is not perfect.
```
Comment (ronkorving) [inline README.md]: *heuristic*? :)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+#### How is it working?
+
+This plugin is only changing Cordova's [webview setting](https://github.com/crosswalk-project/cordova-plugin-crosswalk-webview/blob/master/plugin.xml#L28) dynamically before the creation of the webview.
+
+#### Compability
```
Comment (ronkorving) [inline README.md]: Compatibility

**Comment by tbrebant** [comment]
Updated

**Comment by jrouault** [comment]
Didn't review code thoroughly but overall concept lgtm 👍

**Comment by tbrebant** [comment]
Ok, merging and publishing then.

**Pull request merged**